core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -64,13 +64,15 @@ describe('Translation Modal Component', () => {
       ],
       providers: [
         NgbActiveModal,
+        ContextService,
         {
           provide: ChangeDetectorRef,
           useValue: changeDetectorRef
         }
       ],
       schemas: [NO_ERRORS_SCHEMA]
     }).compileComponents();
+    OppiaAngularRootComponent.contextService = TestBed.get(ContextService);
```

Comment:
TestBed.get is deprecated... TestBed.inject is type safe. Let's use that instead.